Source/Core/DolphinQt/Config/Mapping/WiimoteEmuMotionControlIMU.cpp

```diff
@@ -29,20 +30,19 @@ void WiimoteEmuMotionControlIMU::CreateMainLayout()
 {
   auto* warning_layout = new QHBoxLayout();
   auto* warning_label =
-      new QLabel(tr("WARNING: The controls under Accelerometer and Gyroscope are designed to "
+      new QLabel(tr("The controls under Accelerometer and Gyroscope are designed to "
                     "interface directly with motion sensor hardware. They are not intended for "
                     "mapping traditional buttons, triggers or axes. You might need to configure "
                     "alternate input sources before using these controls."));
   warning_label->setWordWrap(true);
   auto* warning_input_sources_button = new QPushButton(tr("Alternate Input Sources"));
-  warning_layout->addWidget(warning_label, 1);
-  warning_layout->addWidget(warning_input_sources_button, 0, Qt::AlignRight);
+  warning_layout->addWidget(
+      QtUtils::CreateIconWarning(this, QStyle::SP_MessageBoxWarning, warning_label), 1);
+  warning_layout->addWidget(warning_input_sources_button);
   connect(warning_input_sources_button, &QPushButton::clicked, this, [this] {
-    ControllerInterfaceWindow* window = new ControllerInterfaceWindow(this);
-    window->setAttribute(Qt::WA_DeleteOnClose, true);
-    window->setWindowModality(Qt::WindowModality::WindowModal);
```

Comment:
I guess the delete on close was removed because you no longer are dynamically allocating the window.  But why was the modal removed?

Reply:
Not needed with `exec`?  This seems like a strange change to bundle with an icon change PR...

Reply:
`.exec()` makes it modal.

Reply:
Alright, that's what I figured.  Still odd you bundled this.  But I guess it's a cleanup...

Reply:
> This seems like a strange change to bundle with an icon change PR...

The existing code was yucky and I couldn't help myself. :)